job_templates/cyclic_cc_pt/config_fed_client.conf

```diff
@@ -0,0 +1,106 @@
+{
+  # version of the configuration
+  format_version = 2
+
+  # This is the application script which will be invoked. Client can replace this script with user's own training script.
+  app_script = "train.py"
+
+  # Additional arguments needed by the training code. For example, in lightning, these can be --trainer.batch_size=xxx.
+  app_config = ""
+
+  # path to defined PyTorch network
+  model_class_path = "net.Net"
```

Comment:
suggest a bit more explanation as in https://github.com/NVIDIA/NVFlare/blob/main/job_templates/sag_pt/config_fed_server.conf#L11C1-L13C31